docs/user/geojson.rst

```diff
@@ -0,0 +1,223 @@
+|                    |     { "foo" : "bar", "baz" : 10 }                                                 |
++--------------------+-----------------------------------------------------------------------------------+
+| or                 | .. code-block:: json                                                              |
+|                    |                                                                                   |
+|                    |     { "$or" : [ { "foo" : "bar" }, { "baz" : 10 } ] }                             |
++--------------------+-----------------------------------------------------------------------------------+
+
+
+Programmatic Access
+^^^^^^^^^^^^^^^^^^^
+
+The API can be accessed programmatically by creating a GeoTools data store and wrapping it. The module is available
+through maven:
+
+.. code-block:: xml
+
+        <dependency>
+            <groupId>org.locationtech.geomesa</groupId>
+            <artifactId>geomesa-geojson-api_2.11</artifactId>
+            <version>1.3.0</version>
+        </dependency>
+
+Example code in scala:
+
+.. code-block:: scala
+
+    import org.locationtech.geomesa.geojson.{GeoJsonGtIndex, GeoJsonIndex}
+
+    val ds = DataStoreFinder.getDataStore(...)
+    val index: GeoJsonIndex = new GeoJsonGtIndex(ds)
+    index.createIndex("test", Some("$.properties.id"), points = true)
+
+    val features =
+        s"""{ "type": "FeatureCollection",
+           |  "features": [
+           |    {"type":"Feature","geometry":{"type":"Point","coordinates":[30,10]},"properties":{"id":"0","name":"n0"}},
+           |    {"type":"Feature","geometry":{"type":"Point","coordinates":[31,10]},"properties":{"id":"1","name":"n1"}},
+           |    {"type":"Feature","geometry":{"type":"Point","coordinates":[32,10]},"properties":{"id":"2","name":"n2"}}
+           |  ]
+           |}""".stripMargin
+
+    index.add(name, features)
+
+    index.query(name, "{}").toList
+    // result:
+    // {"type":"Feature","geometry":{"type":"Point","coordinates":[30,10]},"properties":{"id":"0","name":"n0"}}
+    // {"type":"Feature","geometry":{"type":"Point","coordinates":[31,10]},"properties":{"id":"1","name":"n1"}}
+    // {"type":"Feature","geometry":{"type":"Point","coordinates":[32,10]},"properties":{"id":"2","name":"n2"}}
+
+    index.query(name, "{}", Map("foo.bar" -> "geometry", "foo.baz" -> "properties.name")).toList
+    // result:
+    // {"foo":{"bar":{"type":"Point","coordinates":[30,10]},"baz":"n0"}}
+    // {"foo":{"bar":{"type":"Point","coordinates":[31,10]},"baz":"n1"}}
+    // {"foo":{"bar":{"type":"Point","coordinates":[32,10]},"baz":"n2"}}
+
+RESTful Access
+^^^^^^^^^^^^^^
+
+The REST endpoint may be installed in GeoServer by extracting the install file into ``geoserver/WEB-INF/lib``:
+
+.. code-block:: bash
+
+    tar -xf geomesa-geojson/geomesa-geojson-gs-plugin/target/geomesa-geojson-gs-plugin_2.11-1.3.0.jar
```

Comment:
Shouldn't this be:

    $ tar -xf geomesa-geojson/geomesa-geojson-gs-plugin/target/geomesa-geojson-gs-plugin_2.11-$VERSION-install.tar.gz -C <dest>